src/app/src/reducers.js

```diff
@@ -1,12 +1,15 @@
 import { createReducer } from 'redux-act';
 import update from 'immutability-helper';
 
-import { updateCount } from './actions'
+import { updateCount } from './actions';
 
 const initialState = {
-    n: 0,
+    n: 0
```

Comment:
I think we should adjust the prettier rules to enforce the trailing comma syntax.